tests/integration/components/o-s-s/badge-test.ts

```diff
@@ -52,6 +52,35 @@ module('Integration | Component | o-s-s/badge', function (hooks) {
     });
   });
 
+  module('skins', function () {
+    test('it throws an error when an unsupported skin is passed', async function (assert: Assert) {
+      setupOnerror((err: Error) => {
+        assert.equal(
+          err.message,
+          '[component][OSS::Badge] Unknown skin. Available skins are: primary, success, alert, error'
+        );
+      });
+
+      await render(hbs`<OSS::Badge @skin="foo" @text="2x" />`);
+    });
+
+    ['primary', 'success', 'alert', 'error'].forEach((skin) => {
```

Comment:
Maybe export the SkinDefinition and iterate directly from it ; this way, when we add another class the test will also be updated 🙂